packages/trader/src/Modules/Trading/Containers/trade.tsx

```diff
@@ -76,6 +78,10 @@ const Trade = observer(() => {
     const [subcategory, setSubcategory] = React.useState<string>();
     const [swipe_index, setSwipeIndex] = React.useState<number | undefined>(0);
 
+    const should_lift_up_navigation =
```

Comment:
perhaps better rename this to should_auto_expand_trade_params or something. Just to match with is_trade_params_expanded